doc/classes/Node.xml

```diff
@@ -84,12 +84,12 @@
 			<return type="void" />
 			<param index="0" name="delta" type="float" />
 			<description>
-				Called during the physics processing step of the main loop. Physics processing means that the frame rate is synced to the physics, i.e. the [param delta] parameter will [i]generally[/i] be constant (see exceptions below). [param delta] is in seconds.
+				Called during the physics processing step of the main loop. Physics processing means that the frame rate is synced to the physics. [param delta] is the time between physics frames in seconds.
```

Comment:
Really hard to understand what is being said here, you should use the word "tick" and not "frame".
I'm not sure what you are in fact trying to say.

Maybe:
"_physics_process is called once on each physics tick, and allows Nodes to synchronize their logic with physics ticks. Delta is the logical time between physics ticks in seconds, and is determined by 1 / ticks_per_sec."